release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 10 June 2024 - Fix - ToC checkbox - Fixed an issue with the ToC URL
```

Comment:
Is there an intranet record for this RN? Also, could you rephrase the RN so it's more clear for the user what exactly the issue was?

Reply:
I forgot to add the RN to this.
The trailing slash was causing the issue, we temporarily removed it before we'll change it to an aka.ms rule.